services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/drs/DRSManagerTest.java

```diff
@@ -0,0 +1,52 @@
+package org.sagebionetworks.repo.manager.drs;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.sagebionetworks.StackConfiguration;
+import org.sagebionetworks.repo.model.drs.ServiceInformation;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+public class DRSManagerTest {
+
+    @Mock
+    DRSManager drsManager;
+    @Mock
+    StackConfiguration stackConfiguration;
+
+
+    @BeforeEach
+    public void before() {
+        when(stackConfiguration.getStack()).thenReturn("dev");
+        when(stackConfiguration.getStackInstance()).thenReturn("417.0.1");
+        drsManager = new DRSManagerImpl(stackConfiguration);
```

Comment:
you should not need a before, as the two when calls will need to move to the test (a junit 5 thing).